plugins/repositories/repositories_renderer.rb

```diff
@@ -20,7 +20,7 @@ def content(description)
     return unless description.repositories
 
     if description.repositories.empty?
-      puts "System has no repositories"
+      puts "System has no repositories."
```

Comment:
I think this could be made consistent with the other messages (`There are no repositories`).